shared/stats/rollup.config.mjs

```diff
@@ -0,0 +1,36 @@
+import { join } from 'path'
+
+import configESM from 'govuk-frontend/rollup.esm.config.mjs'
+import configUMD from 'govuk-frontend/rollup.umd.config.mjs'
+import { paths } from 'govuk-frontend-config'
+import { visualizer } from 'rollup-plugin-visualizer'
+
+/**
+ * Rollup config with stats output
+ *
+ * @param {Record<string, string | undefined>} args - Rollup CLI arguments
+ * @returns {import('rollup').MergedRollupOptions[]} Rollup options
+ */
+export default ({ i: input }) => [
+  configESM({
+    i: input,
+    d: join(paths.stats, 'dist/report/esm')
+  }),
```

Comment:
I'm not sure this sets us up to get the full picture. If I followed correctly, this will only build with `govuk-frontend` as an input, which will be resolved to `all.mjs` only. This means we may miss broken tree shaking like what we had to fix in https://github.com/alphagov/govuk-frontend/pull/2959.

Will this also get us when people import specific components from GOV.UK Frontend?

Reply:
Makes sense. I think I'd lean towards updating the array of configs rather than multiply `npx` calls as that may be easier to review and maintain as we add more components 😊

Reply:
All sorted, I've pushed up an update with [multiple entry examples](https://github.com/alphagov/govuk-frontend/tree/performance-stats/shared/stats/src/javascripts) similar to [your spike options](https://github.com/alphagov/govuk-frontend/pull/3239/files#diff-6b7a7f6278023134eca8e8fa3d757c01528122c05cb8702e834ad9b2ff688a32)

Had to turn off `{ preserveModules }` (now bundling by default) and use `--file` not `--dir` but it works